code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -123,13 +123,14 @@
 					for(var/obj/O in can_see)
 						if(O.anchored)
 							continue
-							if(istype(O, /obj/item) || istype(O, /obj/structure) || istype(O, /obj/machinery))

```

Comment:
Well....that explains...some things.

Reply:
A fix for this has been in #11853 for awhile. I'm kinda surprised you put it in a new PR, as that's gonna generate an unnecessary conflict.